server/api/model/person.js

```diff
@@ -15,13 +15,18 @@ class Person extends BaseModel {
 		return {
 			email: ['required', 'email'],
 			password: ['required', 'string'],
+			name: ['required', 'string'],
+			type: ['required', 'string'],
```

Comment:
will type be saved as the exact label shown to the user at frontend?
also, lets add backend validation for this field so we don't accept just any value